backend/app/Actions/Visits/GetBounceRateChartByDateRangeAction.php

```diff
@@ -22,6 +23,8 @@ public function __construct(ChartVisitRepository $repository)
     public function execute(GetBounceRateChartByDateRangeRequest $request): GetBounceRateChartByDateRangeResponse
     {
         $from = $request->period()->getStartDate();
+        $minStarDate = new Carbon('01/01/2018');
```

Comment:
What is the magic date?

Reply:
instead of '01/01/2018' probably better to take date from the very first element, doesn't it?

Reply:
yes it would be better find the earlier element, but I decided to not complicate